runtime/chxvm_var.cc

```diff
@@ -1,4 +1,5 @@
 #include "runtime/chxvm_var.h"
+#include "runtime/chainerx_util.h"
```

Comment:
Sort alphabetically?

Reply:
I don't know why but this seems to be done by clang-format.

Reply:
:thinking: Then, it's fine. Thanks for your explanation!

Reply:
Ah, now I can see the reason. This was because this .cc file was chxvm_var.cc. Google's style guide says foo.h should come first in foo.cc:

https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

Reply:
The follow up will be done here: https://github.com/pfnet-research/chainer-compiler/pull/306/commits/96f18f5e4345bdfc57e0e2f625f9afbba85033a9